packages/dapp/src/i18n/en.json

```diff
@@ -16,13 +16,17 @@
     "walletMenu": {
       "disconnect": "Disconnect",
       "chains": "Chains",
-      "switchChain": "Switch Chain"
+      "switchChain": "Switch Chain",
+      "copy": "Copy",
+      "explore": "Explore",
+      "copiedMsg": "Wallet address copied"
     },
     "language": {
       "key": "Language",
       "value": "English"
     },
     "themes": {
+      "changeTheme": "Change Theme",
```

Comment:
@chybisov Could you give me a go on this if this works properly with Crowdin? Dont wanna break stuff but I hope I did it right to just add to the English translation (en.json). Thankkkkks! :)